applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_process.py

```diff
@@ -51,33 +83,35 @@ def ExecuteFinalizeSolutionStep(self):
             # Computing forces
             force_coefficient += surface_normal*pressure_coefficient
 
-            # Computing moment
-            mid_point = cond.GetGeometry().Center()
-            lever = mid_point-self.moment_reference_point
-            self.moment_coefficient += _CrossProduct(lever, surface_normal*(-pressure_coefficient))
-
         force_coefficient /= self.reference_area
-        self.moment_coefficient /= self.reference_area
-
-        self.__CalculateWakeTangentAndNormalDirections()
 
         self.lift_coefficient = _DotProduct(force_coefficient,self.wake_normal)
         self.drag_coefficient = _DotProduct(force_coefficient,self.wake_direction)
-
-        self.__ComputeLiftFromJumpCondition()
+        self.lateral_force_coefficient = _DotProduct(force_coefficient,self.span_direction)
 
         KratosMultiphysics.Logger.PrintInfo('ComputeLiftProcess',' Cl = ', self.lift_coefficient)
         KratosMultiphysics.Logger.PrintInfo('ComputeLiftProcess',' Cd = ', self.drag_coefficient)
-        KratosMultiphysics.Logger.PrintInfo('ComputeLiftProcess',' RZ = ', force_coefficient[2])
-        KratosMultiphysics.Logger.PrintInfo('ComputeLiftProcess',' Cm = ', self.moment_coefficient[2])
-        KratosMultiphysics.Logger.PrintInfo('ComputeLiftProcess',' Cl = ' , self.lift_coefficient_jump, ' = ( 2 * DPhi ) / ( U_inf * c )')
+        KratosMultiphysics.Logger.PrintInfo('ComputeLiftProcess',' Cq = ', self.lateral_force_coefficient)
 
         self.fluid_model_part.ProcessInfo.SetValue(CPFApp.LIFT_COEFFICIENT, self.lift_coefficient)
         self.fluid_model_part.ProcessInfo.SetValue(CPFApp.DRAG_COEFFICIENT, self.drag_coefficient)
-        self.fluid_model_part.ProcessInfo.SetValue(CPFApp.MOMENT_COEFFICIENT, self.moment_coefficient[2])
-        self.fluid_model_part.ProcessInfo.SetValue(CPFApp.LIFT_COEFFICIENT_JUMP, self.lift_coefficient_jump)
 
-        self.__ComputeLiftFromFarField()
+    def __ComputeMomentFromPressure(self):
+        self.moment_coefficient = KratosMultiphysics.Vector(3)
+
+        for cond in self.body_model_part.Conditions:
+            surface_normal = cond.GetGeometry().Normal()
+            pressure_coefficient = cond.GetValue(KratosMultiphysics.PRESSURE_COEFFICIENT)
+
+            # Computing moment
+            mid_point = cond.GetGeometry().Center()
+            lever = mid_point-self.moment_reference_point
+            self.moment_coefficient += _CrossProduct(lever, surface_normal*(-pressure_coefficient))
+
+        self.moment_coefficient /= self.reference_area
+
+        KratosMultiphysics.Logger.PrintInfo('ComputeLiftProcess',' Cm = ', self.moment_coefficient[2])
+        self.fluid_model_part.ProcessInfo.SetValue(CPFApp.MOMENT_COEFFICIENT, self.moment_coefficient[2])
 
     def __ComputeLiftFromJumpCondition(self):
```

Comment:
In this process, apart from doing:
```suggestion
    def _ComputeLiftFromJumpCondition(self):
```
Can we also create a method such as "__GetTrailingEdgeNode" (this one can be private) that includes these lines:
https://github.com/KratosMultiphysics/Kratos/blob/5253e5c20272d0557fc8a3f0d81b92a515e9d4d3/applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_process.py#L117-L122

So that we can use the same LiftFromJumpCondition function in the embedded_lift_process (where finding the trailing edge node is different)